Resources/Private/Fusion/Prototypes/Editable.fusion

```diff
@@ -0,0 +1,13 @@
+prototype(PackageFactory.AtomicFusion:Editable) < prototype(PackageFactory.AtomicFusion:Component) {
+	property = null
+	inline = false
+
+	renderer = Neos.Fusion:Tag {
+		tagName = ${props.inline ? 'span' : 'div'}
+		content = ${q(node).property(props.property)}
+		@process.contentElementEditableWrapping = Neos.Neos:ContentElementEditable {
+			property = ${props.property}
+		}
+		@if.hasProperty = ${props.property ? true : false}
```

Comment:
We need a second condition/processor to omit the Tag alltogether, when in Frontend. The integrator did not exactly opt-in to add an extra tag at this point, so we should not force to output it in FE, since it is useless there anyway.